controllers/pvc.go

```diff
@@ -185,41 +185,6 @@ func (r *VolumeSnapshotBackupReconciler) BindPVCToDummyPod(log logr.Logger) (boo
 	return true, nil
 }
 
-func (r *VolumeSnapshotBackupReconciler) buildDummyPod(clonedPVC *corev1.PersistentVolumeClaim, p *corev1.Pod) error {
```

Comment:
No longer building the dummy pod?

Reply:
This build function was never used in the code base.  Additionally, there is nothing that we want to mutate with respect to the dummy pod so build function not necessary IMO.